src/NzbDrone.Common.Test/InstrumentationTests/CleanseLogMessageFixture.cs

```diff
@@ -52,6 +52,12 @@ public class CleanseLogMessageFixture
         [TestCase(@"OutputPath=/home/mySecret/Downloads")]
         [TestCase("Hardlinking episode file: /home/mySecret/Downloads to /media/abc.mkv")]
         [TestCase("Hardlink '/home/mySecret/Downloads/abs.mkv' to '/media/abc.mkv' failed.")]
+        // Qbit Announce URLs
+        [TestCase(@"{""added_on"":1614036190,""amount_left"":0,""auto_tmm"":true,""availability"":-1,""category"":""tv/sonarr-qb"",""completed"":797817900,""completion_on"":1614036256,""content_path"":""/pool/downloads/complete/tv/sonarr-qb/Series.2021.01.18.Bob.Smith.1080p.WEB.h264-BAE.mkv"",""dl_limit"":-1,""dlspeed"":263,""downloaded"":808852634,""downloaded_session"":948852634,""eta"":8640000,""f_l_piece_prio"":false,""force_start"":false,""hash"":""04513h85a6sh18975d999935b2e36a2e55581lol"",""last_activity"":1614036256,""magnet_uri"":""magnet:?xt=urn:btih&dn=Show.2021.01.18.Bob.Smith.1080p.WEB.h264-BAE.mkv&tr=https%3a%2f%2flandof.tv%%2fannounce"",""max_ratio"":-1,""max_seeding_time"":525599,""name"":""Show.2021.01.18.Bob.Smith.1080p.WEB.h264-BAE.mkv"",""num_complete"":49,""num_incomplete"":10,""num_leechs"":0,""num_seeds"":0,""priority"":0,""progress"":1,""ratio"":0,""ratio_limit"":-2,""save_path"":""/pool/downloads/complete/tv/sonarr-qb"",""seeding_time_limit"":-2,""seen_complete"":1614036256,""seq_dl"":false,""size"":797817900,""state"":""stalledUP"",""super_seeding"":false,""tags"":"""",""time_active"":10,""total_size"":797817900,""tracker"":""https://xxx.tv/9pr04sg60killroyimaveql2tyu8xyui/announce"",""trackers_count"":1,""up_limit"":-1,""uploaded"":0,""uploaded_session"":0,""upspeed"":0},")]
```

Comment:
Simplify example.

Also, it never fails since you don't have any of the recognized tokens in there.